### PR TITLE
 #17 Paint array elements which are true

### PR DESCRIPTION
**Pull request by mmorihiro**
fix for #17

### CODE REVIEW

File: core/src/main/com/github/mmorihiro/core/ArrayActor.kt [hunk 1]
```diff
@@ -11,12 +12,19 @@ class ArrayActor : Actor() {
     private val renderer = ShapeRenderer()
     private val size = 10f
 
+    init {
+        array[0][0] = true
+        array[5][5] = true
+        array[10][10] = true
+    }
+
```
Comment (mmorihiro) [inline core/src/main/com/github/mmorihiro/core/ArrayActor.kt:20]: dummy data.

File: core/src/main/com/github/mmorihiro/core/ArrayActor.kt [hunk 2]
```diff
@@ -25,6 +33,16 @@ class ArrayActor : Actor() {
...
         val width = array[0].size * size
         val height = array.size * size
+        renderer.color = Color.GREEN
+        for (i in array.indices) {
+            for (j in array[0].indices) {
+                if (array[j][i]) {
+                    renderer.rect(size * j, size * i, size, size)
+                }
+            }
+        }
```
Comment (mmorihiro) [inline core/src/main/com/github/mmorihiro/core/ArrayActor.kt:43]: This transaction is quite complicated.
Scala's `for` is much better.

File: core/src/main/com/github/mmorihiro/core/ArrayActor.kt [hunk 3]
```diff
@@ -11,12 +12,19 @@ class ArrayActor : Actor() {
...
+    }
+
     override fun draw(batch: Batch?, parentAlpha: Float) {
         batch!!.end()
         renderer.projectionMatrix = batch.projectionMatrix
         renderer.transformMatrix = batch.transformMatrix
         renderer.translate(x, y, 0f)
-        renderer.begin(ShapeRenderer.ShapeType.Line)
+        renderer.begin(ShapeRenderer.ShapeType.Filled)
+        renderer.setAutoShapeType(true)
```
Comment (mmorihiro) [inline core/src/main/com/github/mmorihiro/core/ArrayActor.kt:27]: If I don't set it `true`, renderer throws an `IllegalException` when I change shapetype.

**Review by forestone23 — APPROVED** [review]
LGTM
[LGTM](https://lgtm.in/i/G71LKbMoq)

**Comment by mmorihiro** [comment]
Cute LGTM image. :smile:

**Pull request merged**